apps/site/turbo.json

```diff
@@ -24,7 +25,7 @@
       ]
     },
     "build": {
-      "dependsOn": ["^build"],
+      "dependsOn": ["build-blog-data", "^build"],
```

Comment:
`build-blog-data` is already in `prebuild`, will this cause it to execute twice?

Reply:
surprisingly no 🤔 I wonder if turborepo is smart enough to handle that on its own 🤔

Reply:
Wonderful!